pkg/kubelet/cm/devicemanager/manager_test.go

```diff
@@ -266,7 +267,9 @@ func setupDeviceManager(t *testing.T, devs []*pluginapi.Device, callback monitor
 	originalCallback := w.callback
 	w.callback = func(resourceName string, devices []pluginapi.Device) {
 		originalCallback(resourceName, devices)
+		// fmt.Println("Callback")
```

Comment:
this may be a leftover for debugging, can you please remove them?

Reply:
ok, removed